### PR TITLE
qm-create: add page

### PR DESCRIPTION
**Pull request by Faro1991**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
version 7.2, Wed 04 May 2022 07:30:00 AM CEST

### CODE REVIEW

**Comment by Faro1991** [comment]
related to #8537

File: pages/linux/qm-create.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`qm create {{100}} --start 1`
+
+- Specify the type of operating system on the machine:
+
+`qm create {{100}} --ostype {{win10}}`
+
+- Replace an existing machine (requires archiving it):
+
+`qm create {{100}} --archive {{/path/to/backup_file.tar}} --force 1`
```
Comment (CleanMachine1) [inline pages/linux/qm-create.md:20]:
```suggestion
`qm create {{100}} --archive {{path/to/backup_file.tar}} --force 1`
```

File: pages/linux/qm-create.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Specify a script that is executed on specific triggers during machine lifetime:
+
+`qm create {{100}} --hookscript {{/path/to/execution_script.pl}}`
```
Comment (CleanMachine1) [inline pages/linux/qm-create.md:24]:
```suggestion
`qm create {{100}} --hookscript {{path/to/execution_script.pl}}`
```

File: pages/linux/qm-create.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# qm create
+
+> Create or restore a virtual machine on Qemu/KVM Virtual Machine Manager.
```
Comment (CleanMachine1) [inline pages/linux/qm-create.md:3]:
```suggestion
> Create or restore a virtual machine on QEMU/KVM Virtual Machine Manager.
```

**Review by reinhart1010 — APPROVED** [review]
LGTM to me. Thanks @Faro1991!

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/linux/qm-create.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`qm create {{100}} --ostype {{win10}}`
+
+- Replace an existing machine (requires archiving it):
+
+`qm create {{100}} --archive {{path/to/backup_file.tar}} --force 1`
+
+- Specify a script that is executed on specific triggers during machine lifetime:
+
+`qm create {{100}} --hookscript {{path/to/execution_script.pl}}`
```
Comment (navarroaxel) [inline pages/linux/qm-create.md:24]: any script is made to be executed. right?
```suggestion
`qm create {{100}} --hookscript {{path/to/script.pl}}`
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**